### PR TITLE
support wms

### PR DESCRIPTION
**Pull request by maltaesousa**
Add support for public wms

### CODE REVIEW

File: js/sitnlayers.js [hunk 1]
```diff
@@ -121,11 +125,25 @@
...
+            })
+          })
+        )
+      }
       _map = new ol.Map({
         target: _target,
         layers: [
           sitnLayers.sitnCurrentBaseLayer,
           sitnLayers.sitnDrawLayer,
+          ...sitnLayers.sitnWMSLayers,
```
Comment (kalbermattenm) [inline js/sitnlayers.js:146]: Why are there three dots here?
```suggestion
          sitnLayers.sitnWMSLayers,
```

File: js/sitnlayers.js [hunk 2]
```diff
@@ -110,6 +113,7 @@
     sitnLayers.createMap = function (options) {
       _buttons = options.buttons;
       _baselayers = options.baseLayers;
+      _wmslayers = options.wmslayers
```
Comment (kalbermattenm) [inline js/sitnlayers.js]:
```suggestion
      _wmslayers = options.wmslayers;
```

File: js/sitnlayers.js [hunk 3]
```diff
@@ -121,11 +125,25 @@
...
+        sitnLayers.sitnWMSLayers.push(
+          new ol.layer.Image({
+            extent: _extent,
+            source: new ol.source.ImageWMS({
+              url: _WMSurl,
+              params: { 'LAYERS': _wmslayers.join(',') },
+              serverType: 'mapserver'
+            })
+          })
+        )
```
Comment (kalbermattenm) [inline js/sitnlayers.js]:
```suggestion
        );
```

**Review by kalbermattenm — COMMENTED** [review]
Just some missing (or not, that's up to you) semi-colons and three weird points...

File: js/sitnlayers.js (hunk 1 above)
Comment (maltaesousa) [reply js/sitnlayers.js:146]: This is the spread operator https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Op%C3%A9rateurs/Syntaxe_d%C3%A9composition

Comment (kalbermattenm) [reply js/sitnlayers.js:146]: OK, got it, thanks !

**Pull request merged**